src/Span.h

```diff
@@ -1375,4 +1416,179 @@ void Span::serialize_json(RapidJsonWriter &writer) const
     writer.EndObject();
 }
 
-} // namespace zipkin
\ No newline at end of file
+template <class RapidJsonWriter>
+void Span::serialize_json_v2(RapidJsonWriter &writer) const
+{
+    auto serialize_endpoint = [&writer](const Endpoint *host) {
+        writer.StartObject();
+
+        writer.Key("serviceName");
+        writer.String(host->service_name());
+
+        writer.Key("ipv4");
```

Comment:
ok, it's a bug, fixed in [a2b1401](https://github.com/flier/zipkin-cpp/pull/12/commits/a2b14016591a73b13c033ae7dce727af1736b56e)